src/allowed_args.cpp

```diff
@@ -4,246 +4,525 @@
 
 #include "allowed_args.h"
 
+#include "chainparams.h"
+#include "httpserver.h"
+#include "init.h"
+#include "main.h"
+#include "miner.h"
+#include "net.h"
+#include "policy/policy.h"
+#include "script/sigcache.h"
 #include "tinyformat.h"
+#include "torcontrol.h"
+#include "txdb.h"
+#include "qt/guiconstants.h"
+#include "wallet/wallet.h"
```

Comment:
FWIW the travis tests on BU showed this include needs to be surrounded with #ifdef ENABLE_WALLET.
Otherwise with --disable-wallet I got stuff like:

In file included from allowed_args.cpp:24:
In file included from ./wallet/wallet.h:18:
In file included from ./wallet/walletdb.h:11:
./wallet/db.h:22:10: fatal error: 'db_cxx.h' file not found
#include <db_cxx.h>
         ^
1 error generated.

Don't know if this happens on Classic.